modules/vba_documents/lib/tasks/20230530_scrub_doc_type_from_metadata.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :vba_documents do
+  namespace :data_migration do
+    desc('Scrubs doc_type from the metadata of all UploadSubmission records')
+
+    task scrub_doc_type_from_metadata: :environment do
+      ActiveRecord::Base.connection.execute("
+        UPDATE vba_documents_upload_submissions
+        SET uploaded_pdf = uploaded_pdf::jsonb - 'doc_type';
```

Comment:
I think this is a great approach. The `jsonb` `-` operator is perfect. The only thing I am worried about is the number of records on Sandbox and Prod. 2+ million records will surely give the VM something to think about, especially since it is digging into the complex data structure for each row. What I have done in the past is to batch over the ID's in groups of a thousand or so.